src/components/SocialMedia/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { Fragment } from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import { SocialMediaContainer } from './styles'
+import PropTypes from 'prop-types'
+
+const SocialMedia = ({ customClassName }) => {
+  const data = useStaticQuery(graphql`
+    {
+      SocialMedia: allCosmicjsMedia {
+        nodes {
+          metadata {
+            facebook_link
+            instagram_link
+            youtube_link
+            social_media_links {
+              facebook {
+                url
+              }
+              instagram {
+                url
+              }
+              youtube {
+                url
+              }
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  const facebookLink =
+    data.SocialMedia &&
+    data.SocialMedia.nodes[0] &&
+    data.SocialMedia.nodes[0].metadata &&
+    data.SocialMedia.nodes[0].metadata.facebook_link
+      ? data.SocialMedia.nodes[0].metadata.facebook_link
+      : ''
```

Comment:
Elvis operator at the rescue here.

`
data.SocialMedia &&
    data.SocialMedia.nodes[0] &&
    data.SocialMedia.nodes[0].metadata &&
    data.SocialMedia.nodes[0].metadata.facebook_link
      ? data.SocialMedia.nodes[0].metadata.facebook_link
      : ''
`

becomes

`
    data?.SocialMedia?.nodes?.[0]?.metadata?.facebook_link
      ? data.SocialMedia.nodes[0].metadata.facebook_link
      : ''
`

Check this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

This is valid for all youtube, instagram and media_links too